lib/elixir/lib/inspect/algebra.ex

```diff
@@ -53,7 +58,8 @@ defmodule Inspect.Opts do
             width: 80,
             base: :decimal,
             pretty: false,
-            safe: true
+            safe: true,
+            syntax_colors: false
```

Comment:
We should only insert the reset if there is a color before. So assuming the color implementation is like this:

    if precolor = opts.syntax_colors[identifier] do
      reset = Keyword.get(opts.syntax_colors, :reset, "default for reset")

we should be good to go, no?